src/otk/document.py

```diff
@@ -30,7 +36,10 @@ def __init__(self, path: pathlib.Path, target: str = "", *, warn_duplicated_defs
                 raise OTKError("only target osbuild supported right now")
             self._osbuild_ctx = OSBuildContext(self._ctx)
         state = State()
-        tree = process_include(self._ctx, state, path)
+
+        for path in paths:
```

Comment:
AFAICT what this will do is that only the last file from the commandline will produce a tree, the others will produce vars. 

Which is what we want but it might be surprising for our users, consider:
```
$ otk compile os-pipeline.otk image-pipeline.otk
```
I think this will only produce the image-pipline(?). Maybe adding `--preload-defines` or just `--extra-defines` or something would make this behavior a bit clearer for the user. The implementation would not need to change (except the arg parsing) AFAICT.

Reply:
I was unsure about this too.  I see the appeal of having a list of positional args where the last one is special, but you're right, the explicitness of having a flag for these makes it feel cleaner.

Reply:
Done.